doc/how-to/getting_started_go.rst

```diff
@@ -1,256 +1,279 @@
 .. _getting_started-go:
 
---------------------------------------------------------------------------------
 Connecting from Go
---------------------------------------------------------------------------------
+==================
 
-.. _getting_started-go-pre-requisites:
+**Examples on GitHub**: `sample_db <https://github.com/tarantool/doc/tree/latest/doc/code_snippets/snippets/connectors/instances.enabled/sample_db>`_, `go <https://github.com/tarantool/doc/tree/latest/doc/code_snippets/snippets/connectors/go>`_
 
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-Pre-requisites
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+The tutorial shows how to use the 2.x version of the `go-tarantool <https://github.com/tarantool/go-tarantool>`__ library to create a Go application that connects to a remote Tarantool instance, performs CRUD operations, and executes a stored procedure.
+You can find the full package documentation here: `Client in Go for Tarantool <https://pkg.go.dev/github.com/tarantool/go-tarantool/v2>`__.
 
-Before we proceed:
 
-#. `Install <https://github.com/tarantool/go-tarantool#installation>`__
-   the ``go-tarantool`` library.
 
-#. :ref:`Start <getting_started_db>` Tarantool (locally or in Docker)
-   and make sure that you have created and populated a database as we suggested
-   :ref:`earlier <creating-db-locally>`:
+.. _getting_started_go_sample_db:
 
-   .. code-block:: lua
+Sample database configuration
+-----------------------------
 
-       box.cfg{listen = 3301}
-       s = box.schema.space.create('tester')
-       s:format({
-                {name = 'id', type = 'unsigned'},
-                {name = 'band_name', type = 'string'},
-                {name = 'year', type = 'unsigned'}
-                })
-       s:create_index('primary', {
-                type = 'hash',
-                parts = {'id'}
-                })
-       s:create_index('secondary', {
-                type = 'hash',
-                parts = {'band_name'}
-                })
-       s:insert{1, 'Roxette', 1986}
-       s:insert{2, 'Scorpions', 2015}
-       s:insert{3, 'Ace of Base', 1993}
+..  include:: getting_started_net_box.rst
+    :start-after: connectors_sample_db_config_start
+    :end-before: connectors_sample_db_config_end
 
-   .. IMPORTANT::
+.. _getting_started_go_sample_db_start:
 
-       Please do not close the terminal window
-       where Tarantool is running -- you'll need it soon.
+Starting a sample database application
+--------------------------------------
 
-#. In order to connect to Tarantool as an administrator, reset the password
-   for the ``admin`` user:
+Before creating and starting a client Go application, you need to run the :ref:`sample_db <getting_started_net_box_sample_db>` application using ``tt start``:
```

Comment:
Maybe some link to tutorial on starting the app? 
I expect that some readers may be Go developers that are _complete_ newcomers to Tarantool and literally know nothing about it and tt.

Reply:
added a link to 'tt start'